faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -1453,6 +1453,49 @@ class ClientSpec
     docs("data").to[Seq[Value]].get should have size 10
   }
 
+  it should "create an access provider" in {
+    val name = aRandomString
+    val issuer = aRandomString
+    val jwksUri = "https://db.fauna.com"
```

Comment:
that uri will not work in the future, you can anticipate this and change to something `https://xxxx.auth0.com/`